src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example2.java

```diff
@@ -0,0 +1,47 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="NeedBraces">
+      <property name="tokens" value="LITERAL_IF, LITERAL_ELSE"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.blocks.needbraces;
+
+// xdoc section -- start
+class Example2 {
+  String obj = new String();
+  String value = new String();
+  int counter = 1;
+  int count = 0;
```

Comment:
If you need this to be class members, do this for all examples.
Example code should be same by size and java statements. Only trailing comments are changing.

Reply:
Done !